clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/model/TitusInstance.groovy

```diff
@@ -102,6 +107,14 @@ class TitusInstance implements Instance {
     agentId
   }
 
+  String getIpv4Address() {
+    ipv4Address
```

Comment:
i personally hate the groovy return-less syntax, but for simple getters i am going to say it's just a nit, up to you

Reply:
yep agree , added the return statement